whisper_online.py

```diff
@@ -189,7 +190,7 @@ def ts_words(self, segments):
             if any(s[0] <= start <= s[1] for s in no_speech_segments):
                 # print("Skipping word", word.get("word"), "because it's in a no-speech segment")
                 continue
-            o.append((start, end, word.get("word")))
+            o.append((start, end, word.get("word").strip()))
```

Comment:
What was the issue here? 
Sometimes the spaces inside the words have a meaning. Some words, like "dash-delimited", come in multiple parts, such as " dash"  + "-delimited", so joining with mandatory spaces would break this.

Reply:
I think I saw this makes the output more similar to the faster-whisper backend, but I'm not sure anymore.
In any case; I'll separate this out into another PR if I encounter the issue again.